packages/orchestration/src/utils/zoe-tools.js

```diff
@@ -114,21 +115,24 @@ export const makeZoeTools = (
         throw makeError(`One or more deposits failed ${q(errors)}`);
       }
       // TODO #9541 remove userSeat from baggage
-    },
-  );
+    });
 
   /**
    * Transfer the `amounts` from a `localAccount` to the `recipientSeat`. If any
    * of the withdrawals fail, everything will be rolled back to the
    * `srcLocalAccount`. Supports multiple items in the `amounts`
-   * {@link PaymentKeywordRecord}.
+   * {@link PaymentKeywordRecord}
+   *
+   * @type {HostOf<WithdrawToSeat>}
    */
-  const withdrawToSeat = retriable(
-    zone,
-    'withdrawToSeat',
-    /** @type {WithdrawToSeat} */
-    async (localAccount, destSeat, amounts) => {
-      await null;
+  const withdrawToSeat = (localAccount, destSeat, amounts) =>
+    /*
+     * There are no interchain actions - everything happens on the local chain -
```

Comment:
this explanation is helpful but strikes me as something to document and reference. (It's repeated twice here and surely there are other places equally worth noting it.)

Not a blocker for merge but something we should improve

Reply:
Agree, moved to the `@file` declaration